src/styles/countdown.scss

```diff
@@ -0,0 +1,28 @@
+.countdown {
+  position: fixed;
+  width: 140px;
+  height: 175px;
+  right: 0px;
+  top: 155px;
+
+  background: $light-green;
+  border-radius: 15px 0 0 15px;
+  z-index: 1000;
+
+  &__text-wrapper {
+    margin-top: 40px;
+    margin-left: 25px;
+  }
+
+  &__text {
+    font-size: 18px;
```

Comment:
once we have standardized font styles we might be able to replace these next few styles but lgtm for now!